fiscalyear.py

```diff
@@ -446,10 +452,11 @@ def __ge__(self, other):
             )
 
 
-class FiscalQuarter(object):
+class FiscalQuarter(_Hashable):
     """A class representing a single fiscal quarter."""
 
     __slots__ = ["_fiscal_year", "_fiscal_quarter"]
+    __hash__ = _Hashable.__hash__
```

Comment:
Why do we need to reimplement `__hash__`?

Reply:
Or else it wont inherit from  base class

Reply:
Isn't the whole point of subclasses so that common attributes will be inherited?

Reply:
From python docs:
A class that overrides __eq__() and does not define __hash__() will have its __hash__() implicitly set to None.
If a class that overrides __eq__() needs to retain the implementation of __hash__() from a parent class, the interpreter must be told this explicitly by setting __hash__ = <ParentClass>.__hash__